tests/integration/test_minimum_tls_version.py

```diff
@@ -0,0 +1,43 @@
+from . import integ_test_base
+import os
+
+class TestMinimumTLSVersion(integ_test_base.IntegTestBase):
+    def _get_log_contents(self):
+        with open(self.log_file_path, 'r') as f:
+            return f.read()
+
+    def _get_config_file_name(self, tls_version: str) -> str:
+        config_file = open(os.path.join(self.tmp_dir, "test.conf"), "w+")
+        config_file.write(
+            "[TabPy]\n"
+            "TABPY_PORT = 9005\n"
+            "TABPY_TRANSFER_PROTOCOL = https\n"
+            "TABPY_CERTIFICATE_FILE = ./tests/integration/resources/2019_04_24_to_3018_08_25.crt\n"
+            "TABPY_KEY_FILE = ./tests/integration/resources/2019_04_24_to_3018_08_25.key\n"
+            f"TABPY_MINIMUM_TLS_VERSION = {tls_version}"
+        )
+        pwd_file = self._get_pwd_file()
+        if pwd_file is not None:
+            pwd_file = os.path.abspath(pwd_file)
+            config_file.write(f"TABPY_PWD_FILE = {pwd_file}\n")
+
+        config_file.close()
+        self.delete_config_file = True
+        return config_file.name
+
+class TestMinimumTLSVersionValid(TestMinimumTLSVersion):
```

Comment:
Is there a way to not include the `TABPY_MINIMUM_TLS_VERSION` in the test? In that case should the default get set?